torchvision/transforms/functional.py

```diff
@@ -188,36 +188,6 @@ def to_pil_image(pic, mode=None):
     return Image.fromarray(npimg, mode=mode)
 
 
-def normalize(tensor, mean, std, inplace=False):
```

Comment:
This is a breaking change, because users of `torch.nn.functional.normalize` won't find it anymore.
At least, you should `from .functional_tensor import normalize, erase`

Reply:
Yeah, Got it.

Reply:
Given what I mentioned above, I'm not sure if there is a need to move `normalize` and `erase` to `functional_tensor.py`, because we don't have any implementation based on `PIL` anyway.

Thoughts?

Reply:
Yes but if we want to do something like dispatch from functional -> functional_tensor / functional -> functional_pil, then this might help in ensuring consistency, what do you think ?

Reply:
@fmassa, let me know if I am on same page as you?

Reply:
My thinking is that we don't need to move `normalize` nor `erase` to `functional_tensor.py`.
Users shouldn't need to worry about the existence of `functional_pil` nor `functional_tensor`, just `functional`.

Reply:
Sure, got it ! Thanks